.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [push, pull_request]
```

Comment:
This will trigger on push or pull to _any_ branch – the comment about “master” is false.

I usually trigger on push to main, and pull request to any branch.